lib/src/wallet/traits.rs

```diff
@@ -608,3 +651,37 @@ impl<P: Parameters> DomainWalletExt<P> for OrchardDomain {
         &mut transaction.orchard_notes
     }
 }
+
+pub(crate) trait Diversifiable {
```

Comment:
Why does this look like it's being added for the first time?

Reply:
Well.. Because I think it is!